index.js

```diff
@@ -14,4 +14,17 @@ const maxPage = 1;
 const page = 1;
 const searchQuery = "";
 
-cardContainer.append(newCard);
+async function fetchCharacters() {
+  cardContainer.innerHTML = "";
+  const response = await fetch(
+    "https://rickandmortyapi.com/api/character/?page=1"
+  );
+  const data = await response.json();
+  const allCharacters = data.results;
+  allCharacters.forEach((character) => {
+    const newCharacter = createCharacterCard(character);
```

Comment:
I know, that might seem quiet picky now, but I suggest to pick another variable name here. The information you are saving in your `const` is the created character card. So I would name it something like that (`characterCard`, `newCharacterCard` ...).

Reply:
I like pickiness, so it's all good! 😅